packages/polymath-issuer/src/actions/compliance.js

```diff
@@ -1166,10 +1174,10 @@ export const editManualApproval = (
 ) => async (dispatch: Function, getState: GetState) => {
   const st: SecurityToken = getState().token.token.contract;
   const editingApproval = getState().whitelist.editingApproval;
-  const increase = editingApproval.tokens < allowance ? false : true;
-  const changeAmount = Math.abs(
-    editingApproval.tokens - Web3.utils.fromWei(allowance)
-  );
+  const newAllowance = new BigNumber(Web3.utils.fromWei(allowance).toString());
+  const oldAllowance = new BigNumber(editingApproval.tokens.toString());
+  const increase = newAllowance.isGreaterThan(oldAllowance);
+  const changeAmount = Math.abs(oldAllowance.minus(newAllowance));
```

Comment:
Use `BigNumber` `absoluteValue()` instead of Math.

Reply:
What happens if change amount is zero? Will it still send a transaction?

Reply:
If it's 0 it will still send, as a user maybe changing the date value which is perfectly valid.

Reply:
right!